104/UIKitBasics/UIKitBasics/HomeViewController.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  HomeViewController.swift
+//  UIKitBasics
+//
+//  Created by Rafael Shamsutdinov on 07.10.2022.
+//
+import UIKit
+
+class HomeViewController: UIViewController {
+    let defs = UserDefaults.standard
+    @IBOutlet weak var browseButton: UIButton!
+    @IBOutlet weak var animalButton: UIButton!
+    @IBOutlet weak var name: UILabel!
+    override func viewDidLoad() {
+        name.text = "Hello, \(String(describing: defs.string(forKey: "name")!))"
+        super.viewDidLoad()
+        setConstraints()
+        // Do any additional setup after loading the view.
+    }
+    
+
+    @IBAction func AnimalImages(_ sender: Any) {
+        let vc = storyboard?.instantiateViewController(withIdentifier: "AnimalVC")
+        present(vc!, animated: true)
+    }
+    @IBAction func browseCatalog(_ sender: Any) {
+        let anotherVC: BrowseViewController = UIStoryboard(name: "Main", bundle: nil)
+              .instantiateViewController(identifier: "BrowseViewController")
+        navigationController?.pushViewController(anotherVC, animated: true)
+          }
```

Comment:
Закрывающая скобка должна быть на уровне с местом, где она открывается
```suggestion
    @IBAction func browseCatalog(_ sender: Any) {
        let anotherVC: BrowseViewController = UIStoryboard(name: "Main", bundle: nil)
              .instantiateViewController(identifier: "BrowseViewController")
        navigationController?.pushViewController(anotherVC, animated: true)
    }
```